### DISCUSSION TITLE
BEAMnrc

### DISCUSSION THREAD

**Discussion by mona399** [nrc-cnrc/EGSnrc#1195] [General]
How can simulate guard leaves  and no guard leaves by BEAMnrc??

**Comment by rtownson** [comment]
The BEAMnrc leaf modelling component modules are relatively strict. I'm not sure what guard leaves look like. Are you not able to achieve something similar by adjusting the number of leaves in the CM and setting its cross section?

**Comment by mona399** [comment]
thank you for reply, I'm trying to simulate x MLC 10 cm, Y MLC 12 cm, and Y jaw 10cm (I am using  agility Elekta Linac) What specific component modules must I choose to simulate the MLC can help me to shape the MLC size above

**Reply by rtownson** [reply]
Use SYNCMLCE for modelling Elekta MLCs. You'll need the manufacturer specifications to guide you with how to set the MLC cross section shapes. You can do this using beamnrc_gui.

**Comment by mona399** [comment]
Thank you for the clarification.
